client/login/magic-login/request-login-email-form.jsx

```diff
@@ -78,19 +76,19 @@ class RequestLoginEmailForm extends React.Component {
 				<h1 className="magic-login__form-header">
 					{ translate( 'Email me a login link.' ) }
 				</h1>
-				<LoggedOutForm onSubmit={ this.onSubmit }>
+				<p>{
+					translate( 'Get a link sent to the email address associated ' +
```

Comment:
Nit-picky, but style is more usually:

```js
<p>
    { translate( …
        … ) }
</p>
```

and sometimes with the closing curly on its own line, but personally I think that adds noise.